packages/codeeditor/src/viewer.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { StackedLayout, Widget } from '@lumino/widgets';
+import { CodeEditor, CodeEditorWrapper } from './';
```

Comment:
The practice is to provide the explicit local file for import

```suggestion
import { CodeEditor, CodeEditorWrapper } from './widget';
```

Reply:
again something I copied from widget.ts in the same directory and though was odd, I'll fix it